code/modules/food_and_drinks/restaurant/generic_venues.dm

```diff
@@ -22,8 +22,13 @@
 	var/obj/item/object_to_order = customer_data.get_order(src)
 
 	. = object_to_order
-
-	customer_pawn.say(order_food_line(object_to_order))
+	var/line_to_say = order_food_line(object_to_order)
+	if(prob(1) || SSevents.holidays && SSevents.holidays[APRIL_FOOLS])
+		line_to_say = "Yeah can I get a mothafuckin uh"
```

Comment:
i'm tempted to say leave the 1% normal chance to american customers only, but i trust you know what you're doing